### ISSUE TITLE
Undeclared dependency Guard?

### ISSUE THREAD

**Issue by eserte** [astj/p5-Test-WWW-Stub#14]
My smoker machines started to report failures, maybe due to a missing dependency declaration:
```
#   Failed test 'use Test::WWW::Stub;'
#   at t/00_compile.t line 4.
#     Tried to use 'Test::WWW::Stub'.
#     Error:  Can't locate Guard.pm in @INC (you may need to install the Guard module) (@INC contains: ... .) at /tmpfs/.cpan-build-cpansand/2016031306/Test-WWW-Stub-0.05-MJluqb/blib/lib/Test/WWW/Stub.pm line 9.
# BEGIN failed--compilation aborted at /tmpfs/.cpan-build-cpansand/2016031306/Test-WWW-Stub-0.05-MJluqb/blib/lib/Test/WWW/Stub.pm line 9.
# Compilation failed in require at t/00_compile.t line 4.
# BEGIN failed--compilation aborted at t/00_compile.t line 4.
# Looks like you failed 1 test of 3.
t/00_compile.t .. 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/3 subtests 
```

**Comment by eserte** [comment]
Possibly related: https://metacpan.org/source/MIYAGAWA/LWP-Protocol-PSGI-0.09/Changes#L5

**Comment by astj** [comment]
Thanks for reporting!
I forgot to declare dependency on `Guard`, and it worked with older `LWP::Protocol::PSGI`, as you mentioned.
Now I declared the dependency and released as 0.06.
